### ISSUE TITLE
issues osx 0.4.0 binary

### ISSUE THREAD

**Issue by sdahlbac** [anasinnyk/terraform-provider-1password#15]
I downloaded the 0.4.0, put it in the correct location and when I `terraform plan` I get:

``` 
Error: Error asking for user input: 1 error occurred:
	* provider.onepassword: fork/exec /<redacted>/.terraform/plugins/darwin_amd64/terraform-provider-onepassword_v0.4.0: exec format error
```

Is there a problem with the released version?

**Comment by anasinnyk** [comment]
Yes it's a bug with the release. I miss GO prefix between provider and right now all release working only on Linux. I will fix it in next release after few minutes

**Comment by anasinnyk** [comment]
fixed in release [0.4.1](https://github.com/anasinnyk/terraform-provider-1password/releases/tag/0.4.1) Thanks for the issue.
